pyramid/session.py

```diff
@@ -16,11 +16,12 @@
     text_,
     bytes_,
     native_,
+    urlparse,
     )
 
-from pyramid.exceptions import BadCSRFToken
+from pyramid.exceptions import BadCSRFOrigin, BadCSRFToken
```

Comment:
I would prefer these on multiple lines, with tuple style importing, listed in alphabetical order.